src/components/Search.tsx

```diff
@@ -109,6 +109,7 @@ export function Search({
 }: SearchProps) {
   useDocSearchKeyboardEvents({isOpen, onOpen, onClose});
   return (
+    // @ts-ignore
```

Comment:
What's the error?

Reply:
This was a type mismatch between the v18 react types and the v19 react code. I've since updated React types to v19 and fixed all the downstream type errors.